### PR TITLE
Translate some pages to Chinese

### PR DESCRIPTION
**Pull request by qianmoQ**
Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

Detailed description / Documentation draft:
...

> By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

> If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/zh/development/architecture.md [hunk 1]
```diff
@@ -1,194 +1,199 @@
...
+Read/WriteBuffers只处理字节。`ReadHelpers`和`WriteHelpers`头文件中的函数可以帮助格式化输入/输出。例如，有一些助手可以用十进制格式写一个数字。
 
-## 表（Tables） {#biao-tables}
+让我们看看当您想将`JSON`格式的结果集写入`stdout`时会发生什么。您已准备好从`IBlockInputStream`中获取结果集。您创建`WriteBufferFromFileDescriptor(STDOUT_FILENO)`以将字节写入标准输出。你创建了`JSONRowOutputStream`，用`WriteBuffer`初始化，将`JSON`中的行写入标准输出。你在它上面创建了`BlockOutputStreamFromRowOutputStream`，将它表示为`IBlockOutputStream`。然后你调用`copyData`将数据从`IBlockInputStream`传输到`IBlockOutputStream`。在内部，`JSONRowOutputStream`将编写各种JSON分隔符，并使用对`IColumn`的引用和行号作为参数调用`IDataType::serializeTextJSON`方法。因此，`IDataType::serializeTextJSON`将从`WriteHelpers.h`调用一个方法：例如，`writeText`用于数字类型，`writeJSONString`用于`DataTypeString`。
 
-表由 `IStorage` 接口表示。该接口的不同实现对应不同的表引擎。比如 `StorageMergeTree`、`StorageMemory` 等。这些类的实例就是表。
+## 数据表 {#tables}
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:84]: 感觉这个修改没必要，也不太妥

File: docs/zh/development/architecture.md [hunk 2]
```diff
@@ -1,194 +1,199 @@
...
+还有`TableFunction`。这些函数返回一个临时的`IStorage`对象，用于查询的`FROM`子句。
 
-> 作为 `read` 方法的结果，`IStorage` 返回 `QueryProcessingStage` - 关于 storage 里哪部分查询已经被计算的信息。当前我们仅有非常粗粒度的信息。Storage 无法告诉我们«对于这个范围的数据，我已经处理完了 WHERE 字句里的这部分表达式»。我们需要在这个地方继续努力。
+要快速了解如何实现表引擎，可以看一些简单的东西，比如`StorageMemory`或`StorageTinyLog`。
 
-## 解析器（Parsers） {#jie-xi-qi-parsers}
+> 作为`read`方法的结果，`IStorage`返回`QueryProcessingStage` —— 关于查询的哪些部分已经在存储中计算过的信息。
 
-查询由一个手写递归下降解析器解析。比如， `ParserSelectQuery` 只是针对查询的不同部分递归地调用下层解析器。解析器创建 `AST`。`AST` 由节点表示，节点是 `IAST` 的实例。
+## 解析 {#parsers}
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:105]: 这儿也是

File: docs/zh/development/architecture.md [hunk 3]
```diff
@@ -1,194 +1,199 @@
...
+手写的递归解析器用于解析一个查询。例如，`ParserSelectQuery`只是为查询的各个部分递归调用底层解析器。解析器创建一个`AST`。 `AST`由节点表示，节点是`IAST`的实例。
 
-## 解释器（Interpreters） {#jie-shi-qi-interpreters}
+> 由于历史原因，不使用解析生成器。
 
-解释器负责从 `AST` 创建查询执行流水线。既有一些简单的解释器，如 `InterpreterExistsQuery` 和 `InterpreterDropQuery`，也有更复杂的解释器，如 `InterpreterSelectQuery`。查询执行流水线由块输入或输出流组成。比如，`SELECT` 查询的解释结果是从 `FROM` 字句的结果集中读取数据的 `IBlockInputStream`；`INSERT` 查询的结果是写入需要插入的数据的 `IBlockOutputStream`；`SELECT INSERT` 查询的解释结果是 `IBlockInputStream`，它在第一次读取时返回一个空结果集，同时将数据从 `SELECT` 复制到 `INSERT`。
+## 解释器 {#interpreters}
 
-`InterpreterSelectQuery` 使用 `ExpressionAnalyzer` 和 `ExpressionActions` 机制来进行查询分析和转换。这是大多数基于规则的查询优化完成的地方。`ExpressionAnalyzer` 非常混乱，应该进行重写：不同的查询转换和优化应该被提取出来并划分成不同的类，从而允许模块化转换或查询。
+解析器负责从`AST`创建查询执行管道。简单的解析器，例如`InterpreterExistsQuery`和`InterpreterDropQuery`，或者更复杂的`InterpreterSelectQuery`。查询执行管道是块输入或输出流的组合。例如，解析`SELECT`查询的结果是从`IBlockInputStream`读取结果集； INSERT查询的结果是`IBlockOutputStream`写入数据以进行插入，解析`INSERT SELECT`查询的结果是`IBlockInputStream`，它在第一次读取时返回一个空结果集，但从`SELECT`到`INSERT`同时进行。
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:113]: 解释器

File: docs/zh/development/architecture.md [hunk 4]
```diff
@@ -1,194 +1,199 @@
...
 
-## 函数（Functions） {#han-shu-functions}
+`InterpreterSelectQuery`使用`ExpressionAnalyzer`和`ExpressionActions`机制进行查询分析和转换。这是大多数基于规则的查询优化完成的地方。`ExpressionAnalyzer`非常混乱，应该重写：应该将各种查询转换和优化提取到单独的类中，以允许对查询进行模块化转换。
 
-函数既有普通函数，也有聚合函数。对于聚合函数，请看下一节。
+## 函数 {#functions}
 
-普通函数不会改变行数 - 它们的执行看起来就像是独立地处理每一行数据。实际上，函数不会作用于一个单独的行上，而是作用在以 `Block` 为单位的数据上，以实现向量查询执行。
+有普通函数和聚合函数。有关聚合函数，请参见下一节。
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:119]: 这个修改我觉得也不如之前

File: docs/zh/development/architecture.md [hunk 5]
```diff
@@ -1,194 +1,199 @@
...
 
-还有一些杂项函数，比如 [块大小](../sql-reference/functions/other-functions.md#function-blocksize)、[rowNumberInBlock](../sql-reference/functions/other-functions.md#function-rownumberinblock)，以及 [跑累积](../sql-reference/functions/other-functions.md#function-runningaccumulate)，它们对块进行处理，并且不遵从行的独立性。
+普通函数不会改变行数——它们的工作方式就像独立处理每一行一样。事实上，函数不是针对单个行调用的，而是针对数据的`Block`来实现向量化的查询执行。
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:121]: `针对数据的Block` 很奇怪

File: docs/zh/development/architecture.md [hunk 6]
```diff
@@ -1,194 +1,199 @@
...
+状态在`Arena`(内存池)中分配，以在执行高基数的`GROUP BY`查询时处理多个状态。状态可以有一个非常重要的构造函数和析构函数:例如，复杂的聚合状态可以自己分配额外的内存。它需要注意创建和销毁状态并正确传递它们的所有权和销毁顺序。
 
-> 聚合函数状态的序列化数据格式目前尚未版本化。如果只是临时存储聚合状态，这样是可以的。但是我们有 `AggregatingMergeTree` 表引擎用于增量聚合，并且人们已经在生产中使用它。这就是为什么在未来当我们更改任何聚合函数的序列化格式时需要增加向后兼容的支持。
+聚合状态可以被序列化和反序列化以在分布式查询执行期间通过网络传递或将它们写入没有足够RAM的磁盘上。它们甚至可以存储在带有`DataTypeAggregateFunction`的表中，以允许数据的增量聚合。
 
-## 服务器（Server） {#fu-wu-qi-server}
+> 聚合函数状态的序列化数据格式现在没有版本化。如果聚合状态只是临时存储，则可以。但是我们有用于增量聚合的`AggregatingMergeTree`表引擎，并且人们已经在生产中使用它。这就是将来在更改任何聚合函数的序列化格式时需要向后兼容的原因。
 
-服务器实现了多个不同的接口：
+## 服务 {#server}
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:143]: 服务器

File: docs/zh/development/architecture.md [hunk 7]
```diff
@@ -1,194 +1,199 @@
...
+---
+toc_priority: 62
+toc_title: 架构概述
+---
 
-ClickHouse 是一个真正的列式数据库管理系统（DBMS)。在 ClickHouse 中，数据始终是按列存储的，包括矢量（向量或列块）执行的过程。只要有可能，操作都是基于矢量进行分派的，而不是单个的值，这被称为«矢量化查询执行»，它有利于降低实际的数据处理开销。
+# ClickHouse 架构概述 {#overview-of-clickhouse-architecture}
 
-> 这个想法并不新鲜，其可以追溯到 `APL` 编程语言及其后代：`A +`、`J`、`K` 和 `Q`。矢量编程被大量用于科学数据处理中。即使在关系型数据库中，这个想法也不是什么新的东西：比如，矢量编程也被大量用于 `Vectorwise` 系统中。
+ClickHouse是一个真正面向列的DBMS。数据按列存储，并在执行数组（向量或列块）期间存储。 只要有可能，操作都是在数组上调度的，而不是在单个值上调度。它被称为“向量化查询执行”，它有助于降低实际数据处理的成本。
```
Comment (ucasfl) [inline docs/zh/development/architecture.md:8]: 这儿也不对

**Comment by ucasfl** [comment]
@qianmoQ `architecture.md` 这个文件不是机翻的，不建议重新翻译，很多地方不如之前。

**Comment by vdimir** [comment]
@ucasfl just in case, you can use [suggestion feature](http://haacked.com/archive/2019/06/03/suggested-changes/) on GitHub PR (actually, I think you know about it) to have an  option to quickly commit your suggested changes directly from PR and merge it.

**Comment by vdimir** [comment]
@qianmoQ could you, please, resolve comments from @ucasfl and conflicts with master, and I'll merge PR after that.

**Comment by ucasfl** [comment]
> @qianmoQ could you, please, resolve comments from @ucasfl and conflicts with master, and I'll merge PR after that.

My all comments are about `architecture.md`, this file is not machine translated(I translated it before), and the changes in this file does not better than before, even worse.

**Comment by qianmoQ** [comment]
> @qianmoQ could you, please, resolve comments from @ucasfl and conflicts with master, and I'll merge PR after that.

The file has been reset

File: docs/en/development/continuous-integration.md [hunk 8]
```diff
@@ -206,9 +206,7 @@ Measure changes in query performance. This is the longest check that takes just
 # QA
 
 > What is a `Task (private network)` item on status pages?
```
Comment (vdimir) [inline docs/en/development/continuous-integration.md:208]:
```suggestion
> What is a `Task (private network)` item on status pages?

```

File: docs/en/development/continuous-integration.md [hunk 9]
```diff
@@ -206,9 +206,7 @@ Measure changes in query performance. This is the longest check that takes just
...
-
 It's a link to the Yandex's internal job system. Yandex employees can see the check's start time and its more verbose status.
 
 > Where the tests are run
```
Comment (vdimir) [inline docs/en/development/continuous-integration.md:211]:
```suggestion
> Where the tests are run

```

**Review by vdimir — APPROVED** [review]

**Pull request merged**